.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 
 node_js:
-  - "10.15.1"
-  - "8"
+  - "10.17.0" # lts/dubnium
+  - "12.13.1" # lts/erbium
```

Comment:
can we use "10" and "12" so that we build against the latest minor/patch releases?

Reply:
I think it's better to have it pinned so that our build is stable, otherwise it'll update over time and could start failing?

Reply:
if it fails that might be good to know - it could be failing for users downloading and using that latest version of Node

Reply:
I'm trying to keep our development environments consistent with CI since we don't often work on Kit features, I'd prefer @36degrees make the decision here.

Reply:
The other repositories are not run by our users so I think let's make an exception here and take the risk of it blowing up now and then.

Reply:
I think it's better to know in the (hopefully unlikely) case that a new minor version causes issues with the kit.